### ISSUE TITLE
can't resize window in macos

### ISSUE THREAD

**Issue by mavergit** [trimental/inlyne#177]
Inlyne crashes if i try to resize its window.
Macos Ventura, Inlyne 0.3.2

**Comment by CosmicHorrorDev** [comment]
I'm assuming this is the same issue that I've been seeing locally lately. Here's the crash report that I get for it (ignoring the duplicates since this winds up panicking while panicking)

```toml
"name" = "inlyne"
"operating_system" = "Arch Linux Rolling Release [64-bit]"
"crate_version" = "0.3.2"
"explanation" = """
Panic occurred in file '/home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/wgpu-core-0.17.1/src/resource.rs' at line 207
"""
"cause" = "Map callback was leaked"
"method" = "Panic"
"backtrace" = """

   0: 0x5582515c7a69 - <unresolved>
   1: 0x55825161155a - <unresolved>
   2: 0x5582516ae951 - <unresolved>
   3: 0x558250cfe669 - <unresolved>
   4: 0x558250ce29ee - <unresolved>
   5: 0x558250ce3724 - <unresolved>
   6: 0x558250ce246f - <unresolved>
   7: 0x55825103cbf5 - <unresolved>
   8: 0x558250ccdec3 - <unresolved>
   9: 0x558251032e67 - <unresolved>
  10: 0x7f882a158cd0 - <unresolved>
  11: 0x7f882a158d8a - __libc_start_main
  12: 0x558250ccdc55 - <unresolved>
  13:        0x0 - <unresolved>"""
```

You can see that the crash is happening in `wgpu_core`. It seems like this was already fixed in newer versions, but we're stuck on older versions since `wgpu_glyph` is unlikely to be updated. Looks like we'll have to downgrade the `wgpu` version we're using in v0.3

**Comment by CosmicHorrorDev** [comment]
This should be fixed in the v0.3 branch now. I'll backport some other things and try to release a v0.3.3 tomorrow

**Comment by CosmicHorrorDev** [comment]
This should be fixed with the v0.3.3 release

https://github.com/trimental/inlyne/releases/tag/v0.3.3
